news/_posts/2023-08-08-playtest-20230808.md

```diff
@@ -0,0 +1,56 @@
+---
+kind: article
+title: "Playtest 20230808"
+author: "PunkPun"
+created_at: 2023-08-08 20:00 +0000
+disqus_id: "playtest-20230801"
+permalink: "/news/playtest-20230808/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Another week, another playtest! Notable changes include:
+
+* Added new Red Alert mission "Soviet13b"
+* Fixed dedicated servers not allowing maps with lua to be played
+* Fixed the rare aircraft jitter 
```

Comment:
Rare? Maybe this should also mention that this fixes visual jittering while steering. (I.e. be more formulated from a user POV that they understand what you mean here.)

Reply:
I'm really not sure how word it differently